build_library/toolchain_util.sh

```diff
@@ -333,7 +337,7 @@ install_cross_toolchain() {
         $sudo emerge "${emerge_flags[@]}" \
             "cross-${cross_chost}/gdb" "${cross_pkgs[@]}"
         if [ "${cross_chost}" = aarch64-cros-linux-gnu ] && \
```

Comment:
Is the `cbuild` check needed here, too?

Reply:
I added the cbuild check here for consistency.